common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -973,7 +973,7 @@ public static MtaNotifyClasses fromString(String s) throws ServiceException {
              throw ServiceException.INVALID_REQUEST("invalid value: "+s+", valid values: "+ Arrays.asList(values()), null);
         }
         public boolean isBounce() { return this == bounce;}
-        public boolean is_bounce() { return this == _bounce;}
+        public boolean is_2bounce() { return this == _2bounce;}
```

Comment:
can you please check references of this method. it should not break anything,

Reply:
didn't find any code references for `is_bounce` method